src/main/plugins/org.dita.pdf2/xsl/fo/hazard-domain.xsl

```diff
@@ -0,0 +1,286 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- 20170503 SCH: Add support for troubleshooting elements. -->
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:fo="http://www.w3.org/1999/XSL/Format"
+  xmlns:opentopic="http://www.idiominc.com/opentopic"
+  xmlns:opentopic-index="http://www.idiominc.com/opentopic/index"
+  xmlns:opentopic-func="http://www.idiominc.com/opentopic/exsl/function"
+  xmlns:dita2xslfo="http://dita-ot.sourceforge.net/ns/200910/dita2xslfo"
+  xmlns:ot-placeholder="http://suite-sol.com/namespaces/ot-placeholder"
+  exclude-result-prefixes="ot-placeholder opentopic opentopic-index opentopic-func dita2xslfo xs"
+  version="2.0">
+
+  <!-- hazardstatement =================================================== -->

```

Comment:
This is actually outside of the troubleshooting domain - not sure if it should be located in its own Pull Request, or if we should just broaden the scope of this one so that it's actually "troubleshooting + hazard statement".